compiler/src/dmd/mars.d

```diff
@@ -375,6 +375,8 @@ void setDefaultLibrary(ref Param params, const ref Target target)
 
     if (driverParams.debuglibname is null)
         driverParams.debuglibname = driverParams.defaultlibname;
+    else if (!driverParams.debuglibname.length)  // if `-debuglib=` (i.e. an empty debuglib)
+        driverParams.debuglibname = null;
```

Comment:
This is a slipstreamed fix for `dmd -g -debuglib= foo.d` causing an invalid `/DEFAULTLIB:` linker flag for `link.exe`, which complains about a missing name. The same logic is used for `-defaultlib=` a few lines up.

Reply:
(The Phobos Makefile specifies `-defaultlib= -debuglib=` to make sure no default Phobos library is linked.)